Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -6,7 +6,8 @@
 import json
 import requests
 import dateparser
-
+import uuid
+from MicrosoftApiModule import *  # noqa: E402
```

Comment:
I should leave it here because i'm using this class as base class for `AzureSentinelClient`
and if it will be in the original place `MicrosoftClient` will be "unknown"

Reply:
why did you switch to inheritance instead of composition?
this will be problematic in future bug investigations as the line numbers are not aligned with the codebase.